load.go

```diff
@@ -12,13 +13,19 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/go-playground/mold/modifiers"
```

Comment:
This is going to introduce a dependency to this package on a good half of the Go repositories we have, do you foresee any potential maintenance overhead due to this? (I'm mostly worried about API breaking changes that would result in versions of `segmentio/conf` becoming incompatible with certain versions of `go-playground/mold`.

Reply:
I’ll update the package and PR to use gopkg.in to avoid any potential breaking changes; as much as I’d love to think there will never be breaking changes go-playground/validator has taught me otherwise 😄